src/main/resources/fileTemplates/common_build.gradle.kts.ft

```diff
@@ -56,6 +59,14 @@ kotlin {
                 #else
                 api(compose.material)
                 #end
+
+                #if (${USE_KTOR})
+                implementation("io.ktor:ktor-client-core:$ktorVersion")
```

Comment:
You missed adding a ktor to the web configuration.

Reply:
sorry, I will add this now, thanks for review